code/Importer/StepFile/StepFileImporter.cpp

```diff
@@ -93,7 +93,7 @@ const aiImporterDesc *StepFileImporter::GetInfo() const {
 static const std::string mode = "rb";
 static const std::string StepFileSchema = "CONFIG_CONTROL_DESIGN";
 
-void StepFileImporter::InternReadFile(const std::string &file, aiScene* pScene, IOSystem* pIOHandler) {
+void StepFileImporter::InternReadFile(const std::string &file, aiScene* /*pScene*/, IOSystem* pIOHandler) {
```

Comment:
Just a hint: remove the parametes out of the function definition makes it possible to put bigger regions into comments via /**/.